src/container/srv.c

```diff
@@ -15,10 +15,12 @@
 #include <daos/rpc.h>
 #include "rpc.h"
 #include "srv_internal.h"
+bool ec_agg_disabled;
```

Comment:
Should we also avoid starting the two ULTs for propagating EC upper bound when EC agg is disabled?

Reply:
yes, we should do that too, let me clear it up in the following patch.